index.html

```diff
@@ -6,6 +6,7 @@
     <title>Canonical global navigation demo</title>
     <meta name="description" content="Canonical global navigation demo" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://assets.ubuntu.com/v1/vanilla-framework-version-3.2.0.min.css" />
```

Comment:
This seems couple versions behind

Reply:
Is it needed at all? Global nav project has Vanilla as a dependency.

Reply:
Fixed